packages/devextreme/js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -1415,15 +1428,12 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
     },
 
     _attachClickEvent: function() {
-        const clickSelector = '.' + this._itemClass();
-        const pointerDownSelector = '.' + NODE_CLASS + ', .' + SELECT_ALL_ITEM_CLASS;
-        const eventName = addNamespace(clickEventName, this.NAME);
-        const pointerDownEvent = addNamespace(pointerEvents.down, this.NAME);
         const $itemContainer = this._itemContainer();
+        this._removeClickEvent($itemContainer);
+
+        const { eventName, clickSelector, pointerDownEvent, pointerDownSelector } = this._getEventsData();
 
         const that = this;
```

Comment:
[boy scout rule]
Let's get rid of `that and use arrow functions for events handlers